pkg/context/context.go

```diff
@@ -189,7 +188,8 @@ func NewContext4Test(baseCtx context.Context, withChangefeedVars bool) Context {
 		CaptureInfo: &model.CaptureInfo{
 			ID:            "capture-id-test",
 			AdvertiseAddr: "127.0.0.1:0000",
-			Version:       version.ReleaseVersion,
+			// suppose the current version is `v6.3.0`
+			Version: "v6.3.0",
```

Comment:
Why not `version.ReleaseVersion`?

Reply:
it's `None`, cannot perform the ut easily.